rossum_api/domain_logic/urls.py

```diff
@@ -34,3 +34,31 @@ def build_upload_url(resource: Resource, id_: int) -> str:
 
 def build_full_login_url(base_url: str) -> str:
     return f"{base_url}/auth/login"
+
+
+def build_resource_search_url(resource: Resource) -> str:
+    return f"{resource.value}/search"
```

Comment:
Ok, I will stop commenting on this, the usage of domain logic seem to be lost during the rebase :laughing: